Plugin_SimpleMapDetails.as

```diff
@@ -0,0 +1,61 @@
+#name "SimpleMapDetails"
+#author "derfuhr"
+
+bool windowIsVisible = false;
+
+void Main(){
```

Comment:
You don't need a `Main()` if you have the other functions already doing stuff. :+1: